bioio/tests/test_bio_image.py

```diff
@@ -113,3 +113,28 @@ def test_dump_plugins() -> None:
         subprocess.check_call(
             [sys.executable, "-m", "pip", "uninstall", "-y", package_name]
         )
+
+
+def test_plugin_feasibility_report() -> None:
+    # Arrange
+    test_image = np.random.rand(10, 10)
+    package_name = "dummy-plugin"
+    expected_error_msg = "missing 1 required positional argument: 'path'"
+
+    try:
+        # Install the plugin
+        subprocess.check_call([sys.executable, "-m", "pip", "install", DUMMY_PLUGIN])
+
+        # Act
+        actual_output = bioio.bio_image.plugin_feasibility_report(test_image)
+
+        # Assert
+        assert actual_output["ArrayLike"]["supported"] is True
+        assert actual_output["ArrayLike"]["error"] is None
+        assert actual_output["dummy-plugin"]["supported"] is False
+        assert expected_error_msg in str(actual_output["dummy-plugin"]["error"])
+    finally:
+        # Uninstall the plugin
+        subprocess.check_call(
+            [sys.executable, "-m", "pip", "uninstall", "-y", package_name]
+        )
```

Comment:
This DUMMY_PLUGIN is used in two tests (`test_dump_plugins`) so it seems worth pulling it out into a context manager instead of rewriting the `try..finally`:
```suggestion
# This belongs in another file, not the test file
class InstallPackage:
    def __init__(package_name):
        self.package_name = package_name

    def __enter__(self):
        # Install the plugin
        qualified_package_name = str(pathlib.Path(__file__).parent / self.package_name)
        subprocess.check_call([sys.executable, "-m", "pip", "install", self.package_name])

    def __exit__(self):
        # Uninstall the plugin
        subprocess.check_call(
            [sys.executable, "-m", "pip", "uninstall", "-y", self.package_name]
        )

def test_plugin_feasibility_report() -> None:
    # Arrange
    test_image = np.random.rand(10, 10)
    expected_error_msg = "missing 1 required positional argument: 'path'"

    with InstallPackage("dummy-plugin"):
        # Act
        actual_output = bioio.bio_image.plugin_feasibility_report(test_image)

        # Assert
        assert actual_output["ArrayLike"]["supported"] is True
        assert actual_output["ArrayLike"]["error"] is None
        assert actual_output["dummy-plugin"]["supported"] is False
        assert expected_error_msg in str(actual_output["dummy-plugin"]["error"])
```

Reply:
(This code snippet is untested, do not assume it works)

Reply:
resolved in [0e2c7e7](https://github.com/bioio-devs/bioio/pull/63/commits/0e2c7e751ee25009c648ef79eb29635999686540)